spec/notifiers/publishing_api_notifier_spec.rb

```diff
@@ -9,7 +9,7 @@
       expect(Services.publishing_api).to receive(:put_content).with('69af22e0-da49-4810-9ee4-22b4666ac627', test: :payload)
       expect(Services.publishing_api).to receive(:publish).with('69af22e0-da49-4810-9ee4-22b4666ac627', 'minor')
```

Comment:
1. By _"check that publishing_api receives calls with more than just the original content id"_, do you mean also test the `self.publish` method to ensure it publishes both the start page and at least one of the other pages?

2. In regards to testing that the payloads meet the requirements of a generic schema, I feel the fact that we're already testing for that in the presenters should be enough? As long as the Presenters get instantiated the payload will meet those requirements.